packages/rocketchat-ui-admin/client/rooms/adminRoomInfo.coffee

```diff
@@ -138,12 +138,27 @@ Template.adminRoomInfo.onCreated ->
 						toastr.success TAPi18n.__ 'Room_topic_changed_successfully'
 						RocketChat.callbacks.run 'roomTopicChanged', AdminChatRoom.findOne(rid)
 			when 'roomType'
+				val = @$('input[name=roomType]:checked').val()
 				if @validateRoomType(rid)
 					RocketChat.callbacks.run 'roomTypeChanged', AdminChatRoom.findOne(rid)
-					Meteor.call 'saveRoomSettings', rid, 'roomType', @$('input[name=roomType]:checked').val(), (err, result) ->
-						if err
-							return handleError(err)
-						toastr.success TAPi18n.__ 'Room_type_changed_successfully'
+					func = =>
```

Comment:
could you please rename the `func` variable to anything else that makes more sense?